bin/codecov.ts

```diff
@@ -15,7 +15,7 @@ const realArgs = argv.argv
 
 const start = Date.now()
 
-verbose(`Start of uploader: ${start}...`, Boolean(realArgs.verbose))
+verbose(`Start of uploader: ${start}...`, realArgs.verbose)
```

Comment:
This was being coerced into a Boolean because `tsc` assumed it was a string and was throwing errors. If it now passes without it, 👍